safe_transaction_service/history/indexers/erc20_events_indexer.py

```diff
@@ -71,12 +71,12 @@ def database_field(self):
         return "erc20_block_number"
 
     @property
-    def database_queryset(self):
-        return SafeContract.objects.all()
+    def database_queryset(self) -> QuerySet:
+        return SafeContract.objects
```

Comment:
I think that this could break this https://github.com/safe-global/safe-transaction-service/pull/2079/files#diff-23397faacb994eb17a67a778b0607b0e5739593676738489ae13c9c3433ddfe4R198
Maybe better set the all inside of the queryset.

Reply:
Agree, you already flagged it in another comment